frontends/concrete-python/concrete/fhe/tfhers/context.py

```diff
@@ -0,0 +1,192 @@
+"""
+Declaration of `tfhers.Context` class.
+"""
+
+from typing import List, Optional, Union
+
+from concrete.compiler import TfhersExporter, TfhersFheIntDescription
+
+from concrete import fhe
+
+from .dtypes import EncryptionKeyChoice, TFHERSIntegerType
+
+
+class Context:
```

Comment:
We should indeed find a better naming, is that a Bridge?

Reply:
+1, `Context` is definitely overloaded in the codebase ...

Reply:
I like it. Renamed to Brdige